Login UseCase/Login.java

```diff
@@ -36,6 +34,8 @@ public void setPassword(String password) {
 
 // UserRepository manages user data in a simple in-memory map.
 // For storing and retrieving user information.
+
+
```

Comment:
You Can Check for duplicate users when adding.
You can Use a ConcurrentHashMap for better concurrency if accessed by multiple threads.